tests/Aquifer.Public.API.IntegrationTests/AppFixture.cs

```diff
@@ -0,0 +1,66 @@
+﻿using FastEndpoints.Testing;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Aquifer.Public.API.IntegrationTests;
+
+/// <summary>
+/// This is a FastEndpoints <see cref="AppFixture"/> that sits on top of a <see cref="WebApplicationFactory{TEntryPoint}"/>
+/// which allows for in-memory web requests to be sent and received without actual network traffic.
+/// Details:
+/// * https://fast-endpoints.com/docs/integration-unit-testing
+/// * https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests
+/// </summary>
+public sealed class AppFixture : AppFixture<Program>
```

Comment:
I added a bunch of documentation to this class assuming that we'll copy/paste it for the Internal API (which will need an authenticated client) and eventual Mobile API.